libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -371,7 +474,9 @@ const Row = ({
         dtClassName={className}
         ddClassName={className}
       >
-        {startCase(t(field))}
+        <Tooltip description={tooltipMapping[field]} align="start">
```

Comment:
nitpick (non-blocking): may be nice to have a fallback here if the field is not defined.

Reply:
i think it will just not display a tooltip content in that case, because `tooltipMapping[undefined]` is also going to be `undefined`